src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -0,0 +1,164 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { LogGroupNode } from '../explorer/logGroupNode'
+import * as telemetry from '../../shared/telemetry/telemetry'
+import { CloudWatchAPIParameters, LogStreamRegistry } from '../registry/logStreamRegistry'
+import { convertLogGroupInfoToUri } from '../cloudWatchLogsUtils'
+import { createQuickPick, DataQuickPickItem } from '../../shared/ui/pickerPrompter'
+import { Wizard } from '../../shared/wizards/wizard'
+import { createInputBox } from '../../shared/ui/inputPrompter'
+import { integer } from 'vscode-languageserver-types'
+import { CloudWatchLogsNode } from '../explorer/cloudWatchLogsNode'
+import { AWSTreeNodeBase } from '../../shared/treeview/nodes/awsTreeNodeBase'
+import { AwsContext } from '../../shared/awsContext'
+
+export interface SearchLogGroup {
+    region: string
+    logGroupName: string
+}
+
+export async function searchLogGroup(
+    AWSContext: AwsContext,
+    registry: LogStreamRegistry,
+    node?: LogGroupNode
+): Promise<void> {
+    let result: telemetry.Result = 'Succeeded'
+    const regionOptions = await AWSContext.getExplorerRegions()
```

Comment:
yes, this might also be the right time to implement a more generic `getDefaultRegion()` util function. Currently we only have the credentials-based one, but we need one that draws from recently-touched AWS Explorer items etc (and falls back to credential-default-region).